nh_eobs/views/observation_report_template.xml

```diff
@@ -813,41 +818,10 @@
-                                                                <td class="col-xs-12">
-                                                                    <table class="compact-data col-xs-12">
-                                                                        <tr>
-                                                                            <td colspan="2" class="header col-xs-12">
-                                                                                <h4><t t-if="'summary' in task and task['summary']"><t t-esc="task['summary']"/></t></h4>
-                                                                            </td>
-                                                                        </tr>
-                                                                        <t t-if="'notes' in task and task['notes']">
-                                                                            <tr>
-                                                                                <td colspan="2" class="col-xs-12"><t t-esc="task['notes']"/></td>
-                                                                            </tr>
-                                                                        </t>
-                                                                        <tr>
-                                                                            <td class="col-xs-6"><t t-if="'user_id' in task and task['user_id']"><strong>By:</strong> <t t-esc="task['user_id'][1]"/></t></td>
-                                                                            <td class="col-xs-6"><t t-if="'date_terminated' in task and task['date_terminated']"><strong>Date:</strong> <t t-esc="task['date_terminated']"/></t><br/>
-                                                                                <t t-if="'date_terminated' in task and task['date_terminated']==False">
-                                                                                        Task still in progress
-                                                                                </t>
-                                                                            </td>
-                                                                        </tr>
-                                                                    </table>
-                                                                </td>
-                                                            </tr>
-                                                        </t>
-                                                    </table>
-                                                </td>
-                                            </tr>
+                                            <t t-call="nh_eobs.triggered_actions"/>
+                                        </t>
+                                        <t t-if="'state' in observation and observation['state'] == 'cancelled'">
```

Comment:
Will this separation of full obs and partial (cancelled) obs change the chronological order of the events in the report as currently the report reflects all aspects of the patient journey in this order where as this shows the full obs in chronological order and then the partial (cancelled ) obs in chronological order.

Also what does this mean for actual partial (non-cancelled) obs